.github/workflows/adev-preview-deploy.yml

```diff
@@ -36,43 +36,31 @@ jobs:
           path: ${{ env.BUILD_DIR }}
           github-token: '${{secrets.GITHUB_TOKEN}}'
           run-id: ${{ github.event.workflow_run.id }}
-      - run: ls -R
-      - name: Extract pull request number
-        id: pr-number
-        run: |
-          PR_NUMBER=$(cat ./$BUILD_DIR/__metadata__pull_number.txt)
-          echo "value=$PR_NUMBER" >> $GITHUB_OUTPUT
-      - name: Extract commit hash
-        id: commit-hash
-        run: |
-          COMMIT_HASH=$(cat ./$BUILD_DIR/__metadata__commit_hash.txt)
-          echo "value=$COMMIT_HASH" >> $GITHUB_OUTPUT
-      - run: echo ${{ steps.pr-number.outputs.value }} ${{ steps.commit-hash.outputs.value }}
       - name: Deploy to Firebase Hosting Preview
         id: firebase-deploy
         uses: FirebaseExtended/action-hosting-deploy@v0.9.0
         with:
           repoToken: "${{ secrets.GITHUB_TOKEN }}"
           firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT }}"
           projectId: angular-ja
-          channelId: pr-${{ steps.pr-number.outputs.value }}
+          channelId: pr-${{ github.event.workflow_run.pull_requests.number }}
```

Comment:
I wonder how the line here works because GHA documentation says `workflow_run.pull_requests` is typed as an array of objects. 
https://docs.github.com/en/webhooks/webhook-events-and-payloads#workflow_run

@josephperrott Have you seen other repositories that uses `workflow_run.pull_requests.number` like here?

Reply:
After looking closer into it, it seems that it won't work with this method of extraction, additionally it seems that if a change comes from a fork then it won't provide the pull request at all.  So I reverted back to how it was extracted before.